packages/stream/create-stream/create-stream.js

```diff
@@ -0,0 +1,114 @@
+import { Internal } from '../../internal/index.js'
+import { InitializationInstance } from '../../app/init/init.js'
+
+/**
+ * Function to cek if the input text has a space or not. If yes, then will replace space with "-"
+ *
+ * @function
+ * @param {string} text -- stream's name
+ * @returns {string} -- stream's slug as per stream's name but without space
+ */
+function replaceWhiteSpace(text) {
```

Comment:
This function is to create a slug version based on the stream name right? I found a great thread about creating a slug from a string here https://gist.github.com/codeguy/6684588?permalink_comment_id=4176055#gistcomment-4176055. 
Maybe we can use this function to slugify the string

```js
function slugify(text) {
  return text
    .toString()                           // Cast to string (optional)
    .normalize('NFKD')            // The normalize() using NFKD method returns the Unicode Normalization Form of a given string.
    .toLowerCase()                  // Convert the string to lowercase letters
    .trim()                                  // Remove whitespace from both sides of a string (optional)
    .replace(/\s+/g, '-')            // Replace spaces with -
    .replace(/[^\w\-]+/g, '')     // Remove all non-word chars
    .replace(/\-\-+/g, '-')        // Replace multiple - with single -
    .replace(/\-$/g, ''); // Remove trailing -
}
```

Reply:
> This function is to create a slug version based on the stream name right?

@faiq-naufal yups mas that's right, because I think it will be weird if no slug (in case in further development we will use this slug too).
Such a good function mas, thank you for the reference! Already done at commit [7548c46](https://github.com/inlivedev/inlive-js-sdk/pull/11/commits/7548c46a0baae8b053ca3ba4700b73a453372dcc)